themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -12,18 +12,12 @@ VuFind.register('itemStatuses', function ItemStatuses() {
     return callnumber;
   }
   function displayItemStatus(result, $item) {
-    $item.removeClass('js-item-pending');
```

Comment:
Can you elaborate on why we don't want to remove this class? I gather this is because we want to short-circuit our way out of checkItemStatus under certain circumstances, but I don't exactly follow why, and I wonder if there's a clearer way to do it, given that with this change, things will remain pending even after they are technically no longer in a pending state.

Reply:
We need to keep it (or rename it to something like 'js-item-done' to indicate that the item has been processed. Ths makes sure that an item is never processed twice even if item statuses are initialized twice for the container or Hunt fires the event multiple times for a single item. We might be able to eliminate both situations, but I believe it's a good practice to still guard against accidental refetch.

Reply:
How do you feel about https://github.com/vufind-org/vufind/pull/1466/commits/796c34c3d6002b3c5d3d32f3524cf655a7e7750a? I think this more clearly expresses what is going on. Also, if I'm reading the code correctly, if we leave the pending statuses in place, we run the risk of a situation where a single request failing in isolation causes all of the status indicators on the screen to get overwritten with error messages. (I think the error handling is less than ideal in any case, but we don't need to make it even worse). :-) I'm also interested in @crhallberg's thoughts!

Reply:
796c34c works for me. Thanks!